lib/internal/crypto/cipher.js

```diff
@@ -154,8 +154,20 @@ ObjectSetPrototypeOf(Cipher.prototype, LazyTransform.prototype);
 ObjectSetPrototypeOf(Cipher, LazyTransform);
 
 Cipher.prototype._transform = function _transform(chunk, encoding, callback) {
-  this.push(this[kHandle].update(chunk, encoding));
-  callback();
+  let error = null;
+  try {
+    if (typeof chunk === 'string') {
+      if (!encoding) encoding = getDefaultEncoding();
+      validateEncoding(chunk, encoding);
+    } else if (!isArrayBufferView(chunk)) {
+      throw new ERR_INVALID_ARG_TYPE(
+        'chunk', ['string', 'Buffer', 'TypedArray', 'DataView'], chunk);
+    }
```

Comment:
We should not validate that `chunk` is a string or ArrayBufferView here, that's the streams implementation's job